datahub-web-react/package.json

```diff
@@ -104,11 +102,7 @@
         "ec2-dev": "yarn run generate && CI=true;export CI;BROWSER=none craco start",
         "build": "yarn run generate && CI=false REACT_APP_MOCK=false craco build && rm -rf dist/ && cp -r build/yarn/ dist/ && rm -r build/yarn/",
         "test": "craco test",
-        "cy:run:ci": "cypress run",
         "pretest:e2e:ci": "yarn generate",
-        "test:e2e:ci": "start-server-and-test start:e2e 3010 cy:run:ci",
```

Comment:
we call this out in datahub-web-react's README - same thing with `test:e2e` below.

I think we can update the README so that we just have the `yarn test:e2e` but update `test:e2e` to not call `cy:open`

Reply:
so we can still get rid of `test:e2e:ci` since it will be the same since we're getting rid of the cypress commands

Reply:
thanks for calling that out. Cleaned up README file.